src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -76,10 +76,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.basic());
+
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
Does this still allow @hengelman's work to display styled text work?

Reply:
For some reasons, the changes she made to the code isn't reflecting in my system

Reply:
This won't work with the styled text it needs to be .basic or all HTML will be disallowed

Reply:
Yea I changed it back to basic in my last commit 
dd19931